packages/ui/src/lib/WalletMultiButton.svelte

```diff
@@ -5,6 +5,8 @@
 	import WalletModal from './WalletModal.svelte';
 	import './styles.css';
 
+	export let maxNumberOfWallets = 3;
```

Comment:
@andrewjmeier I think if we define here the amount of wallets, we should remove that number inside the `<WalletModal />` component. 

`export let maxNumberOfWallets;`